sagemaker-python-sdk/tensorflow_distributed_mnist/tensorflow_batch_transform_mnist.ipynb

```diff
@@ -136,7 +138,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### SageMaker's transformer class\n",
+    "## SageMaker's transformer class\n",
     "\n",
     "After training, we use our TensorFlow estimator object to create a `Transformer` by invoking the `transformer()` method. This method takes arguments for configuring our options with the batch transform job; these do not need to be the same values as the one we used for the training job.\n",
```

Comment:
Might be worth mentioning here that this method also creates sagemaker models.

Reply:
done